README.md

```diff
@@ -54,7 +54,7 @@ What does the seed give you out of the box?
 |demo/tests| The tests for your plugin|
 |src| The plugin source code|
 |src/platform/android| Plugin Android specific configuration|
-|src/platform/ios|Plugin ios specific configuration|
+|src/platform/ios|Plugin ios specific configuration. All configurations that are applicable via XCode Info tab and are saved in application's Info.plist file can also be applied for plugins. |
```

Comment:
Since this is a capability of NativeScript plugins in general rather then some feature of the `nativescript-plugin-seed` package I think that it will be better to include this sentence in [this article](https://docs.nativescript.org/plugins/plugin-reference#directory-structure) under the `platforms\ios\Info.plist` section.